Fix for Meteor 1.3

trajano opened the pull request:
Fixes https://github.com/Urigo/angular-blaze-template/issues/14

trajano committed:
Fix for Meteor 1.3

Fixes https://github.com/Urigo/angular-blaze-template/issues/14

Urigo commented:
@trajano does your fix works also with Meteor versions 1.2 and older?
look also here: http://guide.meteor.com/1.3-migration.html#angular-meteor-packages

trajano commented:
No I changed it to be 1.3 only with all the imports and exports.  Is there a way of packaging in Meteor so I can do

if (Meteor.versionAtLeast("1.3") { use mine} else { use existing? }

Urigo commented:
yes, look at the link I've sent

trajano commented:
OK I'll add that part in, but would it still work with 1.2 is the question, I am using the `pbastowski:angular-babel` package

trajano committed:
Enable Meteor 1.2 support

Per https://github.com/Urigo/angular-blaze-template/pull/16#issuecomment-219280708

Urigo commented:
@trajano have you tested it with both Meteor versions?

trajano commented:
No, I don't have Meteor 1.2 on my system anymore.

Urigo commented:
it's easy to run from version, sorry, would need you to test that before I can merge

trajano commented:
NP would just need to find time to do it.

trajano commented:
Actually I have to remove that last commit, those changes added back "angular" to Meteor causing the #14 anyway.

trajano committed:
Revert "Enable Meteor 1.2 support"

This reverts commit 37e488fc5f1935f41a20ffcd24577a7b38ff684a.

1.2 brings back angular packages which causes
https://github.com/Urigo/angular-blaze-template/issues/14 in the first
place.

trajano committed:
Repackaged to trajano:angular-blaze-template
